README.md

```diff
@@ -189,16 +247,15 @@ space.enter({
   avatar: 'https://slides-internal.com/users/clemons.png',
 });
 
+// Listen to events published on "mousemove" by all members
+space.cursors.subscribe('update', (cursorUpdate) => {
```

Comment:
this only works with `space.cursors.subscribe("cursorsUpdate", ... ` & not as `space.cursors.subscribe("update",...`

@surminus @Srushtika

Reply:
@lmars @surminus looks like it's a bug as it's something we agreed on. It's also possible that it slipped the implementation part as there were quite a few changes - are you able to check this please?https://ably.atlassian.net/wiki/spaces/product/pages/2659713025/MMDR14+Spaces+SDK+New+API+Design#Member-cursors

Reply:
https://github.com/ably-labs/spaces/pull/120